src/platform/utilities/api/index.js

```diff
@@ -3,6 +3,21 @@ import * as Sentry from '@sentry/browser';
 import environment from '../environment';
 import localStorage from '../storage/localStorage';
 
+function refreshSSOeSession() {
+  // TODO replace arbitrary resource fetch with more intentional enpoint
+  fetch('https://eauth.va.gov/favicon.ico', {
```

Comment:
This probably needs to be parameterized per-environment, so dev goes to int.eauth.va.gov, etc.

Reply:
adding environment mappings. there's also `http://preprod.eauth.va.gov/`, but I don't think there's an analogue for that on our side